src/shared/hooks/use-lara-interactive-api.ts

```diff
@@ -35,15 +35,19 @@ export const useLARAInteractiveAPI = <AS, IS>(config: IConfig) => {
   const handleInteractiveStateChange = (state: any) => {
     setInteractiveState(state);
     phone.current?.post("interactiveState", state);
-  }
+  };
 
   const handleSetHeight = (height: number) => {
     phone.current?.post("height", height);
-  }
+  };
+
+  const handleSetHint = (hint: string) => {
+    phone.current?.post("hint", hint);
+  };
```

Comment:
This is something we'll have to work out in the lara interactive client, but it seems possible that the hint message could be posted to LARA before supportedFeatures is sent to LARA. I guess that is OK, but we should clarify whether that is allowed or not.